js/home.js

```diff
@@ -0,0 +1,5 @@
+$(document).ready(function() {
```

Comment:
Document ready is not required if you are including the js file at the end of the file